.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: java
 jdk: oraclejdk8
-sudo: false
```

Comment:
Doesn't help 😢

Reply:
Yeah - reverted - I was just comparing with kafka and this seemed like the only difference.